__tests__/app.test.js

```diff
@@ -28,8 +28,62 @@ describe('GET /api/categories', () => {
             }))
         })
     });
+
+    test('200 status: GET /api/reviews return array of objects containing relevant properties, sorted by date in descending order by default', () => {
+        return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({body}) => {
+            const reviews = body.reviews;
+          
+
+            expect(Array.isArray(reviews)).toBe(true);
+            expect(reviews).toHaveLength(13);
+            reviews.forEach((review) => {
+                expect(review).toHaveProperty("owner");
+                expect(review).toHaveProperty("title");
+                expect(review).toHaveProperty("review_id");
+                expect(review).toHaveProperty("category");
+                expect(review).toHaveProperty("review_img_url");
+                expect(review).toHaveProperty("created_at");
+                expect(review).toHaveProperty("votes");
+                expect(review).toHaveProperty("designer");
+                expect(review).toHaveProperty("comment_count");
+            })
+            
+            expect(reviews).toBeSortedBy("created_at", {descending : true,});
+            
+            expect(reviews[4].comment_count).toBe("3"); //test needs updating if testData is changed. NB app express exports a JSON, hence expecting 3 as "3"
+
+            
+          
+        })
+    });
+
+    test('200: /api/reviews/:review_id returns an object specific to that id with  all the relevant key:value pairs', () => {
+        return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({body}) =>{
+                const reviewObj = body.reviewObj[0]
+    
+                expect(typeof reviewObj).toBe("object");
+                expect(Array.isArray(reviewObj)).toBe(false);
+                
+                expect(reviewObj).toHaveProperty("review_id");
+                expect(reviewObj).toHaveProperty("title");
+                expect(reviewObj).toHaveProperty("review_body");
+                expect(reviewObj).toHaveProperty("designer");
+                expect(reviewObj).toHaveProperty("review_img_url");
+                expect(reviewObj).toHaveProperty("votes");
+                expect(reviewObj).toHaveProperty("category");
+                expect(reviewObj).toHaveProperty("owner");
+                expect(reviewObj).toHaveProperty("created_at");
+            })
```

Comment:
Have a look at. more flexible matcher like expect.objectContaining or .toMatchObject for this test